app/templates/share.js

```diff
@@ -99,6 +138,21 @@ module.exports = function(state, emit) {
   </div>
   `;
 
+  if (div.querySelector('#reset-form'))
+    div.querySelector('#reset-form').onsubmit = resetPassword;
+
+  function resetPassword(event) {
+    event.preventDefault();
+    const existingPassword = document.querySelector('.passwordOriginal')
+      .innerText;
```

Comment:
This is regarding indentation . Shall I do this 
 ```javascript
const existingPassword = document.querySelector('.passwordOriginal')
        .innerText;
```
or this
```javascript
 const existingPassword = document.querySelector('.passwordOriginal').innerText;
```

Reply:
In general try to keep it on one line, but our auto-formatter `prettier` might change it.